packages/connect-node/src/grpc-protocol.ts

```diff
@@ -270,21 +268,6 @@ export function createGrpcProtocol(
               );
             }
 
-            if (
-              requestCompression === undefined &&
-              inputResult.value.flags !== messageFlag
-            ) {
-              return await endWithGrpcTrailer(
-                res,
-                context,
-                new ConnectError(
-                  `Unexpected input flags ${inputResult.value.flags.toString(
-                    2
-                  )}`,
-                  Code.Internal
-                )
-              );
-            }
```

Comment:
I was wondering about this for a while and didn't see the point. I think I introduced it 🙈 😆